Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_Faster_RCNN.md

# Convert ONNX Faster R-CNN Model {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_Faster_RCNN}

> Below instructions are applicable **only** to the Faster R-CNN model converted to the ONNX file format from the [maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark):

Author: ```suggestion
The instructions below are applicable **only** to the Faster R-CNN model converted to the ONNX file format from the [maskrcnn-benchmark model](https://github.com/facebookresearch/maskrcnn-benchmark):
```

> 1. Download the pre-trained model file from [onnx/models](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/faster-rcnn):

Author: ```suggestion
1. Download the pretrained model file from [onnx/models](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/faster-rcnn):
```

   * (commit-SHA: 8883e49e68de7b43e263d56b9ed156dfa1e03117).

> 2. Generate the models Intermediate Representation, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:

Author: ```suggestion
2. Generate the model Intermediate Representation, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:
```

or

```suggestion
2. Generate the Intermediate Representation of the model, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:
```

Inanimate objects do not take possessive. It's better to use an adjective or "of" phrase.
I second the latter.

> 2. Generate the Intermediate Representation of the model, by changing your current working directory to the Model Optimizer installation directory, and running the Model Optimizer with the following parameters:

Reviewer: ```suggestion
2. Generate the Intermediate Representation of the model, by changing your current working directory to the Model Optimizer installation directory, and running Model Optimizer with the following parameters:
```

--transformations_config front/onnx/faster_rcnn.json
```

Be aware that the height and width specified with the `input_shape` command line parameter could be different. Refer to the [Faster R-CNN article](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/faster-rcnn) for more information about supported input image dimensions and required pre- and post-processing steps.

> 3. Interpret the outputs. The generated IR file has several outputs from the *`DetectionOutput`* layer: 

Author: ```suggestion
3. Interpret the outputs of the generated IR: class indices, probabilities and box coordinates. Below are the outputs from the "DetectionOutput" layer:
```
I would stray from the idea of creating a list here.

>    * class indices.

Author: ```suggestion
```
